tasks.py

```diff
@@ -10,10 +10,9 @@
 from invoke import task
 
 
```

Comment:
since invoke can be also triggered from a subdirectory:

```
from pathlib import Path

ROOT = Path(__file__).parent.resolve()
os.chdir(ROOT)
```

Reply:
Added.